src/folio_migration_tools/mapping_file_transformation/mapping_file_mapping_base_impl.py

```diff
@@ -29,12 +28,4 @@ def __init__(
         )
 
     def get_prop(self, legacy_item, folio_prop_name, index_or_id):
-        legacy_item_keys = self.mapped_from_legacy_data.get(folio_prop_name, [])
-
-        if len(legacy_item_keys) == 1 and folio_prop_name in self.mapped_from_values:
-            return self.mapped_from_values.get(folio_prop_name, "")
-
-        legacy_values = MappingFileMapperBase.get_legacy_vals(legacy_item, legacy_item_keys)
-        if len(legacy_item_keys) > 1:
-            self.migration_report.add(Blurbs.Details, f"{legacy_item_keys} were concatenated")
-        return " ".join(legacy_values).strip()
+        return super().get_prop(legacy_item, folio_prop_name, index_or_id)
```

Comment:
Curious why this method is still here if all it does is call the same method on the parent class?

Reply:
Good point. It is just confusing. The point of the class was to test a "clean" implementation of get_prop.